tests/analyzer/sourcecode/dll-hijacking.py

```diff
@@ -0,0 +1,135 @@
+"""Injecting a DLL into a process"""
```

Comment:
Can we add some references to usage in the wild? e.g. https://www.reversinglabs.com/blog/attackers-leverage-pypi-to-sideload-malicious-dlls
https://blogs.jpcert.or.jp/en/2024/02/lazarus_pypi.html

I assume your examples are based onthis

Reply:
Sure.
Mind that some techniques are based on posts references but not every case is.